styles.css

```diff
@@ -3,9 +3,10 @@
   margin: 0;
 }
 
-body {
+.section-wrap {
   font-family: "Poppins", sans-serif;
   background-color: #f7f7f9;
+  width: 375px;
 }
```

Comment:
- [ Required ] I think it would be better if the wrapper section for the projects is displayed using the `CSS` grid property as requested by the previous reviewer. The CSS `display: grid` makes it easy for a developer to create a webpage that is responsive across different screen sizes. Please see the screenshot of the project requirements below and rectify this.

## Example code
```CSS
.section-wrap {
  font-family: "Poppins", sans-serif;
  background-color: #f7f7f9;
  width: 375px;
  display: grid;
  grid-template-columns: auto;
}

```
## Screeenshot
 ![Screenshot from 2023-07-13 02-05-24](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/16392046/3c22a7fb-68e7-45c0-b3a1-8285be77c615)